src/proxies_v8/proxies_v8.ts

```diff
@@ -985,6 +985,10 @@ proxiesV8.use('/competencySubTheme/*',
   proxyCreatorSunbird(express.Router(), `${CONSTANTS.KONG_API_BASE}`)
 )
 
+proxiesV8.use('/halloffame/*',
```

Comment:
Please check I think halloffame is already there

Reply:
yes halloffame api's are there but prefix was not there